frontend/components/Pill.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const Pill = styled.div`
```

Comment:
Pills should probably have a higher font weight as well, and a little padding on the right and left